requests-for-comments/2024-02-13-web-analytics-launch.md

```diff
@@ -0,0 +1,64 @@
+## Web analytics marketing launch plan
+
+The launch date for web analytics isn't nailed down yet, but it's _seeming_ like it'll be in March. Either way, it's never too early to plan for how we'll market the heck out it. 
+
+It's worth noting that Web Analytics has been in opt-in beta for some time too, so we can rely on this group of beta-testing customers as an additional launch asset!
+
+We have some [new best practices for product launches around templates](https://posthog.com/handbook/engineering/development-process#best-practices-for-full-releases), but these mostly do not apply to web analytics due to its scope. 
+
+## Pricing?
+
+Pricing is TBC, but web analytics will have usage-based pricing. 
+
+## Site updates
+
+Basic stuff across the site that'll need to be done. 
+
+- [ ] Update documentation - @andyvan https://github.com/PostHog/posthog.com/issues/7736
+- [ ] Create a product page 
+- [ ] Web analytics case study - @joethreepwood
+- [ ] Remove beta tags from homepage slider, docs, etc - @joethreepwood
+- [ ] Add Web analytics links to footer - @joethreepwood
+- [ ] Update `/blog/ga4-alternatives` with a CTA
+- [ ] Update `/blog/track-your-website-with-posthog` with a CTA
+- [ ] Update `/blog/google-analytics-to-posthog` with a CTA
+- [ ] Update `https://posthog.com/blog/posthog-vs-ga4` with a CTA
+- [ ] Create a Web Analytics section for `/questions`
+- [ ] Add a Frequently Asked Questions post to community - @joethreepwood
+
+## Other updates
+
+Basic stuff outside the website that we'll need to update. 
+
+- [ ] Add Web Analytics as an option in in-app onboarding - GROWTH
```

Comment:
I think it'll be part of product analytics, see https://github.com/PostHog/product-internal/pull/566

Reply:
As in it'll be included with product analytics, or some sort of sub-choice from there? 

I'd like to have a web analytics path in the onboarding emails, but in order to do that I'd ideally need users to make an explicit choice to focus on it.

Reply:
tbd, will update